python/lsst/ts/maintel/standardscripts/mtdome/crawl_az.py

```diff
@@ -70,20 +71,29 @@ def get_schema(cls):
                     type: string
                     default: ClockWise
                     enum: ["ClockWise", "CounterClockWise"]
+                position:
+                    description: Target azimuth (in degrees) to slew the dome to before crawling (optional).
+                    type: number
+                velocity:
```

Comment:
You may want to restrict this to be > 0 since the direction is intended to be set by the enum above. As it is, a user could input -0.5 and "CounterClockWise" which would actually go in the opposite direction.